app/views/docs/functions.phtml

```diff
@@ -93,25 +93,54 @@ $runtimes = $this->getParam('runtimes', []);
     <p><b>Python</b></p>
 
     <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-        <pre class="line-numbers"><code class="prism language-bash" data-prism>PIP_TARGET=./.appwrite pip install -r ./requirements.txt --upgrade --ignore-installed</code></pre>
+        <pre class="line-numbers">
+            <code class="prism language-bash" data-prism>PIP_TARGET=./.appwrite </code>
+            <code class="prism language-bash" data-prism>pip install -r ./requirements.txt --upgrade --ignore-installed</code>
+        </pre>
+    </div>
+    <div class="ide margin-bottom" data-lang="powershell" data-lang-label="PowerShell">
+        <pre class="line-numbers">
+            <code class="prism language-bash" data-prism>$env:PIP_TARGET=.appwrite </code>
+            <code class="prism language-bash" data-prism>pip install -r ./requirements.txt --upgrade --ignore-installed</code>
```

Comment:
The powershell part is a good addition. Let's keep it a single line command like so 

```suggestion
            <code class="prism language-bash" data-prism>$env:PIP_TARGET=.appwrite ; pip install -r ./requirements.txt --upgrade --ignore-installed
</code>
```

Reply:
Please reflect the similar **one liner command** in all of your suggestions

Reply:
Sure,

It can be a one liner in powershell